heartbeat/heartbeat.full.yml

```diff
@@ -119,6 +135,22 @@ heartbeat.monitors:
     # Required TLS protocols
     #supported_protocols: ["TLSv1.0", "TLSv1.1", "TLSv1.2"]
 
+  # The tags of the monitors are included in their own field with each
```

Comment:
I suggest we add it only to one monitor (first one) as an example and not every single one.